File: Cinemabe/Cinema/Data/Models/FoodAndDrink.cs
namespace Cinema.Data.Models
{
    public class FoodAndDrink
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Image { get; set; }
        public double Price { get; set; }
        public Guid TheaterId { get; set; }
        public Theater Theater { get; set; }
        public bool Status { get; set; }
    }
}

File: cinema_app/lib/data/models/food_and_drink.dart
class FoodAndDrink {
  String id;
  String name;
  String description;
  String image;
  int price;
  int quantity;

  FoodAndDrink(
      {this.description = "",
      this.id = "",
      this.image = "",
      this.name = "",
      this.price = 0, this.quantity=0});

  FoodAndDrink.fromJson(Map<String, dynamic> json)
      : id = json["id"] ?? "",
        name = json["name"] ?? "",
        description = json["description"] ?? "",
        image = json["image"] ?? "",
        quantity=0,
        price = json["price"] ?? 0;
}
